src/core/components/layout-utils.jsx

```diff
@@ -190,7 +193,16 @@ export class Select extends React.Component {
     let value = this.state.value?.toJS?.() || this.state.value
 
     return (
-      <select className={this.props.className} multiple={ multiple } value={value} onChange={ this.onChange } disabled={disabled} aria-label={description}>
+      <select
+        className={this.props.className}
+        multiple={ multiple }
+        value={value}
+        onChange={ this.onChange }
```

Comment:
small nit: don't think we need the spaces for before / after the prop e.g. they should be like how it's passed in `value`

Reply:
It seems we (and the upstream project!) are mixing the usage of these two styles everywhere, Iet me change it here and leave the rest as is 😮‍💨